packages/wrangler/src/__tests__/pages/deploy.test.ts

```diff
@@ -4808,15 +5110,15 @@ Failed to publish your Function. Got error: Uncaught TypeError: a is not a funct
 
 		it("should bundle the _worker.js when both `--bundle` and `--no-bundle` are omitted", async () => {
 			simulateServer((generatedWorkerJS) =>
-				expect(workerIsBundled(generatedWorkerJS)).toBeTruthy()
+				expect(workerIsBundled(generatedWorkerJS)).resolves.toBeTruthy()
```

Comment:
Hmm. This `expect()` call now returns a Promise.

Reply:
We await it here: https://github.com/cloudflare/workers-sdk/blob/1c8f981711bcc9d0430b47bf584460e4865f1505/packages/wrangler/src/__tests__/pages/deploy.test.ts#L5058